src/pet.js

```diff
@@ -0,0 +1,82 @@
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_HUNGER = 0;
+const FITNESS_THRESHOLD = 3;
+const HUNGER_THRESHOLD = 5;
+const AGE_LIMIT = 30;
+const HUNGER_LIMIT = 10;
+const FITNESS_LIMIT = 0;
+
+Pet.prototype = {
+    get isAlive() {
+        return (this.age < AGE_LIMIT) && (this.hunger < HUNGER_LIMIT) && (this.fitness > FITNESS_LIMIT);
+    }
+};
+
+function Pet(name){
+    this.name = name;
+    this.age = 0;
+    this.hunger = 0;
+    this.fitness = 10;
+    this.children = [];
+    this.growUp = function(){
+        if (!this.isAlive){
+            throw new Error('Your pet is no longer alive :(')
+        }
+        this.age += 1;
+        this.hunger += 5;
+        this.fitness -= 3;
+    };
+    this.walk = function(){
```

Comment:
Adding a clear line between each function makes things a bit easier to read. This could be done for all the functions here
```suggestion

    this.walk = function(){
```